indico/modules/events/surveys/forms.py

```diff
@@ -43,7 +43,7 @@ class SurveyForm(IndicoForm):
     anonymous = BooleanField(_("Anonymous submissions"), widget=SwitchWidget(),
                              description=_("User information will not be attached to submissions"))
     require_user = BooleanField(_("Only logged-in users"), [HiddenUnless('anonymous')], widget=SwitchWidget(),
-                                description=_("Still require users to be logged in for submitting the survey"))
+                                description=_("Require users to be logged in for submitting the survey"))
```

Comment:
IIRC the reason for the "still" was that this only applies to anonymous surveys - i.e. the submission is anonymous, but the user needs to be logged in anyway. But since we hide the option if the survey is not anonymous I guess it's clear without the "Still" (and it indeed sounds better without)